titanic.ipynb

```diff
@@ -5,14 +5,16 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 Line 15_
```python
plt.style.use('dark_background')

# Create a countplot of the man/woman/child classification
sns.countplot(data=titanic, x='who', hue='alive', palette='bright')

# Add some labels and title
plt.title('Survival Rates by Man/Woman/Child')
plt.xlabel('Person Type')
```
my commet

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/25)._